crates/cxx-qt-gen/test_outputs/inheritance.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#include <memory>
+#include <mutex>
+
+namespace rust::cxxqtlib1 {
+template<typename T>
+class CxxQtThread;
+}
+
+class MyObject;
+using MyObjectCxxQtThread = ::rust::cxxqtlib1::CxxQtThread<MyObject>;
+
+#include "cxx-qt-gen/inheritance.cxx.h"
+
+class MyObject : public QAbstractItemModel
+{
+  Q_OBJECT
+
+public:
+  explicit MyObject(QObject* parent = nullptr);
+  ~MyObject();
+  MyObjectRust const& unsafeRust() const;
+  MyObjectRust& unsafeRustMut();
+  ::std::unique_ptr<MyObjectCxxQtThread> qtThread() const;
+
+public:
+  Q_INVOKABLE QVariant data(QModelIndex const& _index,
+                            ::std::int32_t _role) const override;
+  Q_INVOKABLE bool hasChildren(QModelIndex const& _parent) const override;
+  template<class... Args>
+  bool hasChildren_cxxqt_inherit(Args... args) const
+  {
+    return QAbstractItemModel::hasChildren(args...);
+  }
+  template<class... Args>
+  void fetchMore_cxxqt_inherit(Args... args)
```

Comment:
should these be camelcase instead of snake ?

Reply:
Fair point, will update.

Reply:
Done